### ISSUE TITLE
Weird output with every action

### ISSUE THREAD

**Issue by tgentet-obrien** [forcedotcom/cli#929]
### Summary
With any command I am getting the following output:

```shell
sfdx force:org:open
(node:88385) Warning: Accessing non-existent property 'cat' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
(node:88385) Warning: Accessing non-existent property 'cd' of module exports inside circular dependency
(node:88385) Warning: Accessing non-existent property 'chmod' of module exports inside circular dependency
(node:88385) Warning: Accessing non-existent property 'cp' of module exports inside circular dependency
(node:88385) Warning: Accessing non-existent property 'dirs' of module exports inside circular dependency
(node:88385) Warning: Accessing non-existent property 'pushd' of module exports inside circular dependency
(node:88385) Warning: Accessing non-existent property 'popd' of module exports inside circular dependency
(node:88385) Warning: Accessing non-existent property 'echo' of module exports inside circular dependency
(node:88385) Warning: Accessing non-existent property 'tempdir' of module exports inside circular dependency
(node:88385) Warning: Accessing non-existent property 'pwd' of module exports inside circular dependency
(node:88385) Warning: Accessing non-existent property 'exec' of module exports inside circular dependency
(node:88385) Warning: Accessing non-existent property 'ls' of module exports inside circular dependency
(node:88385) Warning: Accessing non-existent property 'find' of module exports inside circular dependency
(node:88385) Warning: Accessing non-existent property 'grep' of module exports inside circular dependency
(node:88385) Warning: Accessing non-existent property 'head' of module exports inside circular dependency
(node:88385) Warning: Accessing non-existent property 'ln' of module exports inside circular dependency
(node:88385) Warning: Accessing non-existent property 'mkdir' of module exports inside circular dependency
(node:88385) Warning: Accessing non-existent property 'rm' of module exports inside circular dependency
(node:88385) Warning: Accessing non-existent property 'mv' of module exports inside circular dependency
(node:88385) Warning: Accessing non-existent property 'sed' of module exports inside circular dependency
(node:88385) Warning: Accessing non-existent property 'set' of module exports inside circular dependency
(node:88385) Warning: Accessing non-existent property 'sort' of module exports inside circular dependency
(node:88385) Warning: Accessing non-existent property 'tail' of module exports inside circular dependency
(node:88385) Warning: Accessing non-existent property 'test' of module exports inside circular dependency
(node:88385) Warning: Accessing non-existent property 'to' of module exports inside circular dependency
(node:88385) Warning: Accessing non-existent property 'toEnd' of module exports inside circular dependency
(node:88385) Warning: Accessing non-existent property 'touch' of module exports inside circular dependency
(node:88385) Warning: Accessing non-existent property 'uniq' of module exports inside circular dependency
(node:88385) Warning: Accessing non-existent property 'which' of module exports inside circular dependency
```

### Steps To Reproduce:

**Repository to reproduce:** [dreamhouse-lwc](https://github.com/dreamhouseapp/dreamhouse-lwc)

***NOTE:** If your issue is not reproducable by dreamhouse-lwc, i.e. requires specific metadata or files, we **require** a link to a simple Salesforce project repository with a script to setup a scratch org that reproduces your problem.*

1.  Run a command
2. 
### Expected result

Not the additional output

### Actual result

The output mentioned above

### Additional information
![image](https://user-images.githubusercontent.com/6325514/111643180-a67f5780-87f6-11eb-8b83-bd7773210a0e.png)

sfdx-cli/7.91.0 darwin-x64 node-v14.16.0

@oclif/plugin-autocomplete 0.3.0 (core)
@oclif/plugin-commands 1.3.0 (core)
@oclif/plugin-not-found 1.2.4 (core)
@oclif/plugin-plugins 1.10.0 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/analytics 0.23.0
@salesforce/lwc-dev-server 2.10.0
├─ @oclif/plugin-update 1.3.9
└─ @oclif/plugin-help 2.2.3
@salesforce/sfdx-diff 0.0.6
@salesforce/sfdx-scanner 2.6.0
@salesforce/sfdx-trust 3.6.0 (core)
alias 1.1.8 (core)
auth 1.4.10 (core)
config 1.2.6 (core)
generator 1.1.5 (core)
salesforcedx 51.3.0 (core)
├─ schema 1.0.4 (core)
├─ limits 1.0.4 (core)
├─ user 1.1.2 (core)
├─ apex 0.1.4 (core)
├─ templates 51.3.0 (core)
├─ @salesforce/sfdx-plugin-lwc-test 0.1.7 (core)
├─ custom-metadata 1.0.11 (core)
└─ salesforce-alm 51.3.0 (core)
sfdx-cli 7.91.0 (core)
telemetry 1.1.1 (core)

**OS and version**:
macOS Catalina v10.15.6

**Comment by MuriloKakazu** [comment]
It seems like a problem with the `lwc-dev-server` plugin https://github.com/forcedotcom/lwc-dev-server-feedback/issues/106

If you're not using it, try:
```console
$ sfdx plugins:uninstall @salesforce/lwc-dev-server
```

Otherwise, I'm not sure of any workarounds... but it seems like it's a known problem (from the issue above on the other repo)

**Comment by tgentet-obrien** [comment]
@MuriloKakazu uninstalling then reinstalling seemed to fix it. No idea why 🙃

**Comment by steals** [comment]
reinstall doesn't work for me. Stil have the same warnings:

> @oclif/plugin-autocomplete 0.3.0
@oclif/plugin-commands 1.3.0 (core)
@oclif/plugin-not-found 1.2.4 (core)
@oclif/plugin-plugins 1.10.0 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/lwc-dev-server 2.10.0
├─ @oclif/plugin-update 1.3.10
└─ @oclif/plugin-help 2.2.1
@salesforce/sfdx-diff 0.0.6
@salesforce/sfdx-trust 3.6.0 (core)
alias 1.1.8 (core)
auth 1.5.1 (core)
config 1.2.7 (core)
etcopydata 0.5.7
generator 1.1.5 (core)
isvte-sfdx-plugin 1.1.11
salesforcedx 51.6.0 (core)
├─ limits 1.0.4 (core)
├─ data 0.4.3 (core)
├─ schema 1.0.4 (core)
├─ org 1.6.0 (core)
├─ custom-metadata 1.0.11 (core)
├─ apex 0.1.4 (core)
├─ templates 51.3.0 (core)
├─ @salesforce/sfdx-plugin-lwc-test 0.1.7 (core)
├─ user 1.2.1 (core)
└─ salesforce-alm 51.6.6 (core)
sfdmu 4.0.0
sfdx-cli 7.94.3 (core)
sfdx-falcon 0.0.93
sfdx-git-delta 4.2.3
sfdx-git-packager 0.0.0 (link) /Users/steals/Work/sfdx-git-packager
sfdx-migration-automatic 3.0.0
sfpowerkit 2.8.6
shane-sfdx-plugins 4.43.0
├─ @mshanemc/sfdx-sosl 1.1.0
└─ @mshanemc/plugin-streaming 1.1.7
telemetry 1.1.1 (core)
